Containerfile

```diff
@@ -19,21 +19,16 @@ ARG IMAGE_NAME="${IMAGE_NAME:-silverblue}"
 ARG FEDORA_MAJOR_VERSION="${FEDORA_MAJOR_VERSION:-40}"
 ARG KERNEL_VERSION="${KERNEL_VERSION:-6.9.7-200.fc40.x86_64}"
 
-COPY sys_files/usr /usr
+COPY system_files/usr /usr
 
 RUN --mount=type=cache,dst=/var/cache/rpm-ostree \
-    --mount=type=bind,from=ctx,src=/,dst=/ctx \
+    --mount=type=bind,rw=true,from=ctx,src=/,dst=/ctx \
```

Comment:
why does is this bind mount changed to `rw=true`?

Reply:
This was added because I got annoyed with permissions on the scripts/ directory.  The Containerfile now `chmod +x`s all scripts in the directory, which required write access.   It can be removed if we wish to proceed with this.

Reply:
I like the bind being read-only, seems a bit odd to mutate the scripts on the fly.

Alternatives are:
1. chmod +x the scripts locally and commit the change 
2. `bash /ctx/build_files/scriptname.sh` at runtime instead of `/ctx/build_files/scriptname.sh`

Reply:
Option 2 seems solid, if all the scripts are standardized as bash scripts.

Reply:
I find it unlikely for us to move off of bash any time soon so #2 would be reasonable for me.

Reply:
I don't really understand why the permissions are problematic in the first place. They are stored in git repo.

Only potential problem is someone committing in windows (not WSL).